public/index.html

```diff
@@ -88,7 +58,29 @@ <h1 id="title">LaTeX Online</h1>
 <a href="https://github.com/aslushnikov/latex-online#ccommand-line-interface">Tell
 me more</a>
                         </li>
+                    </ol>                    
+                    <h3>Parameters</h3>
```

Comment:
(sorry for the caps in the previous post - unfortunate markup!)

Regarding the H3:
- visually it's not distinct enough to make the point, looks more like a glitch
- hierarchical nesting is unfortunate in the current layout

let's rather have H2 here